docs/tasks/administer-cluster/kubeadm-upgrade-1-8.md

```diff
@@ -202,6 +202,12 @@ $ kubeadm upgrade apply v1.8.0
    find your CNI provider and see if there are additional upgrade steps
    necessary.
 
+6. Add RBAC permissions for automated certificate rotation. In the future, kubeadm will perform this step automatically:
+
+```shell
+kubectl create clusterrolebinding autoapprove-node-client-rotation --clusterrole=system:certificates.k8s.io:certificatesigningrequests:selfnodeclient --group=system:nodes
```

Comment:
This role won’t exist until after you upgrade the apiserver to 1.8, so need to move this step after the api server upgrade.

Reply:
It is after the control plane upgrade. Step 4 is the `kubeadm upgrade apply` command that upgrades the control plane.

Reply:
Ah, saw the following section named “upgrading your master”

Reply:
@luxas were you expecting to grant to the whole group or individual nodes?

Reply:
Yes, it is expected to grant the whole group the right to autoapprove their credentials.